src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import {
+  Strategy as GoogleStrategy
+} from 'passport-google-oauth20';
+import dotenv from 'dotenv';
+import {
+  passportCb
+} from '../utils/helpers';
+
+dotenv.config();
+
+const FacebookStrategy = require('passport-facebook').Strategy;
```

Comment:
Why are you using `require` here?. Is it an ES6 syntax?. Please adhere to using only ES6 syntax hence forth.